lib/java_buildpack/framework/dynatrace_one_agent.rb

```diff
@@ -191,7 +200,11 @@ def logstream?
       end
 
       def skip_errors?
-        credentials[SKIP_ERRORS].to_b
+        credentials[SKIP_ERRORS] == "true"
```

Comment:
why ignoring ` credentials[SKIP_ERRORS]` ?

Reply:
sorry my mistake ; should be the same - why the change though?

Reply:
We found that the method was not working in the way we expected it (see https://try.ruby-lang.org/playground/#code=class+String%0A++def+to_b%0A++++casecmp+'true'%0A++end%0Aend%0A%0Aif+%22false%22.to_b%0A++puts+%22yes%22%0Aend&engine=cruby-3.2.0).

Also, from my understanding it would match against "t" or other parts of "true". Which contradicts the information in our documentation.

Reply:
ok, makes sense!